apps/gitness/src/pages/repo/repo-summary.tsx

```diff
@@ -55,7 +55,7 @@ export const RepoSummary: React.FC = () => {
   const { data: readmeContent } = useGetContentQuery({
     path: 'README.md',
     repo_ref: repoRef,
-    queryParams: { include_commit: false, git_ref: normalizedGitRef }
+    queryParams: { include_commit: false, git_ref: normalizeGitRef(selectedBranch) }
```

Comment:
Same here, refetch on branch selection.

Reply:
@vardanbansal-harness  that is taken care of since the state var selectedBranch is passed in queryParams